client/src/canvas/ViewSidebar.res

```diff
@@ -1136,7 +1137,7 @@ let viewSidebar_ = (m: model): Html.html<msg> => {
   | _ => false
   }
 
-  let showAdminDebugger = if !isDetailed && m.isAdmin {
+  let showDarkDevDebugger = if isDetailed && m.editorSettings.showDarkDevDebugger {
```

Comment:
I'm not sure why this was `!isDetailed` instead of `isDetailed`. Intuitively (and in practice, locally), we only want to show the debugger when the sidebar is expanded.